settings.gradle.kts

```diff
@@ -26,15 +27,15 @@ dependencyResolutionManagement {
 }
 
 val isCI = System.getenv("CI") != null
-val skipBuildscan = System.getenv("SKIP_BUILDSCAN").toBoolean()
+val publishBuildscan = System.getenv("PUBLISH_BUILDSCAN").toBoolean()
 gradleEnterprise {
+  server = "https://ge.opentelemetry.io"
   buildScan {
-    termsOfServiceUrl = "https://gradle.com/terms-of-service"
-    termsOfServiceAgree = "yes"
+    publishAlwaysIf(isCI || publishBuildscan)
```

Comment:
```suggestion
    publishAlways()
    publishIfAuthenticated()
```

Reply:
@runningcode Do we need both `always` and `ifAuthenticated`?

Reply:
Yes, its undocumented but this is the recommended set up for OSS projects.